### ISSUE TITLE
Provide a deb repository that can be indexed in apt for auto-updates

### ISSUE THREAD

**Issue by Kreyren** [ProtonMail/proton-bridge#199]
Currently i have to update the bridge manually which is pita

**Comment by aradapilot** [comment]
+1 this would be great, it's the only package these days that requires manual download/installation and patching, seems like a risk

**Comment by andrzejsza** [comment]
yes, that's definitely something that we would like to do and it has been in the backlog for a while. For internal reference: GODT-1100.

**Comment by ploum** [comment]
If possible, it would be nice to have it in the same repository than protonvpn repository. That way, we could simply add one repository and have everything protonrelated in one place.

**Comment by semiviral** [comment]
Any update on this sort of functionality? It would be a very, very convenient way to deploy the different offerings from Proton.

**Comment by tangodev** [comment]
+1. This would be a great QOL feature for sysadmins.
